README.md

```
@@ -1,152 +1,158 @@
+![image](https://github.com/user-attachments/assets/ecc94303-df23-46ae-87dc-66b040396e0b)
+
+If you have a suggestion that would make this better, please fork the repo and
+create a pull request. You can also
+[open issues](https://github.com/superset-sh/superset/issues).
+
+See the [CONTRIBUTING.md](CONTRIBUTING.md) for instructions and code of conduct.
+
+#### Contributors
+
+<a href="https://github.com/superset-sh/superset/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=superset-sh/superset" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)